src/components/AdminForm.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+
+const API_BASE_URL = 'https://dev-kosj-api.fly.dev';
```

Comment:
Never use hard coded variables such as this one. Please use the boilerplate already in development
- Update your branch with the development
- Checkout an example of exercises pages, where I create an exercise server (interface between UI and API, some sort of business logic middleware)